GameServerLib/Content/CharData.cs

```diff
@@ -25,33 +25,6 @@ public class GlobalData : IGlobalData
         public float PercentEXPBonusMaximum { get; set; } = 5.000f;
     }
 
-    public class PassiveData : IPassiveData
```

Comment:
Keep PassiveData, you can leave the `GetPassiveAbilityNameFromScriptFile` function removed for now, but keep the `TODO` comment.